content/visualize/visualize_1.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Visualize your data"
+weight = 1
+post = ""
++++
+
+#### Introduction
+
+Now that you have modeled data you would probably want to see the results in a visual way as well to be able to easily identify any performance issues with your website.
+
+We have created a Tableau workbook to showcase how you could do that, using csvs that were exported from the derived core web vitals tables (`core_web_vitals`, `core_web_vital_measurements`) with [this sample csv](add link to S3) as the data source.
+
+If you would like to see your own modeled data you can simply change the data sources and then you will be able to look at the visualizations to get inspiration of what you can build with your connector of choice (based on the warehouse you have).
+
+#### Accessing the sample Tableau workbook
+
+You can download the file from here:
+
+Once you have the file locally, you can open it in several ways: with Tableau Public, Tableau Reader (free) or by uploading it to Tableau Online, if you have access.
+
+#### Change the dashboard data source
+
+1. Export the `derived.snowplow_web_vitals` and `snowplow_web_vital_measurements` as csv with your sql editor of choice.
+
+2. In Tableau Public or Tableau Online open the sample twbx file and unhide one of the worksheets (E.g. Go to Core Web Vitals dashboard and select LCP by period and device then right click and uncheck the `Hide` box)
+
+3. Go to `Data/ New Data Source` select `text file` and navigate to the downloaded `snowplow_web_vital_measurements.csv`. Go back to the unhidden worksheet. Now you can see the new data source added at the top left corner.
+
+4. Select it and make the following changes:
+
+ - change the `Time Period` to `Date and Time` by clicking on the `Abc` icon in front of its name
+ - create set by right-clicking on `Device Class` then choose `Create / Set`, tick `all` and tick the box `exclude` to exclude results with `all` in the name, click ok
+
+5. Go to `Data / Replace data source` and select the new `snowplow_web_vital_measurements` file to replace `snowplow_web_vital_measurements_sample` in the `Replacement` section.
+
+6. You can right click and `Close` the original `snowplow_web_vital_measurements_sample` data connection from the upper `Data` section of the page.
+
+7. Add and replace the `snowplow_web_vitals_sample` as well. For that you do not need any changes to the fields. (Step 3, 5, 6 are needed again)
+
+You should have a working dashboard with your own data! Feel free to pick and choose visualizations and dashboards to make your custom one.
+
+![Chrome Extension](../images/1.png?width=50pc)
```

Comment:
Alt text is wrong, also could you try just embedding the dashboard? `<div class='tableauPlaceholder' id='viz1686834908764' style='position: relative'><noscript><a href='#'><img alt=' ' src='https:&#47;&#47;public.tableau.com&#47;static&#47;images&#47;Sn&#47;SnowplowCoreWebVitals&#47;LowperfrormingURLs&#47;1_rss.png' style='border: none' /></a></noscript><object class='tableauViz'  style='display:none;'><param name='host_url' value='https%3A%2F%2Fpublic.tableau.com%2F' /> <param name='embed_code_version' value='3' /> <param name='path' value='views&#47;SnowplowCoreWebVitals&#47;LowperfrormingURLs?:language=en-US&amp;:embed=true' /> <param name='toolbar' value='yes' /><param name='static_image' value='https:&#47;&#47;public.tableau.com&#47;static&#47;images&#47;Sn&#47;SnowplowCoreWebVitals&#47;LowperfrormingURLs&#47;1.png' /> <param name='animate_transition' value='yes' /><param name='display_static_image' value='yes' /><param name='display_spinner' value='yes' /><param name='display_overlay' value='yes' /><param name='display_count' value='yes' /><param name='language' value='en-US' /></object></div>                <script type='text/javascript'>                    var divElement = document.getElementById('viz1686834908764');                    var vizElement = divElement.getElementsByTagName('object')[0];                    if ( divElement.offsetWidth > 800 ) { vizElement.style.width='1000px';vizElement.style.height='850px';} else if ( divElement.offsetWidth > 500 ) { vizElement.style.width='1000px';vizElement.style.height='850px';} else { vizElement.style.width='100%';vizElement.style.height='3900px';}                     var scriptElement = document.createElement('script');                    scriptElement.src = 'https://public.tableau.com/javascripts/api/viz_v1.js';                    vizElement.parentNode.insertBefore(scriptElement, vizElement);                </script>`

Reply:
The tableau generated code does not work, but the accelerator template now contains the code to be able to render it: {{< tableau name="SnowplowCoreWebVitals/CoreWebVitals?publish=yes" />}}